.github/workflows/trivy.yml

```diff
@@ -35,6 +35,7 @@ jobs:
 
       # build the image
       - name: Build
+        continue-on-error: true
```

Comment:
Hmm.. what does this accomplish?

Reply:
This is just me testing the GHA, please ignore I will remove it soon